### ISSUE TITLE
how to set warpctc?

### ISSUE THREAD

**Issue by changss** [kasyoukin/caffe_ocr_for_linux#5]
I1213 20:59:11.824182 27507 layer_factory.hpp:77] Creating layer ctcloss
F1213 20:59:11.824225 27507 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: WarpCTCLoss (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, CTCGreedyDecoder, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, DenseBlock, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, Interp, LRN, LSTM, LSTMUnit, Log, Lstm, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, RNN, ReLU, Reduction, Reshape, Reverse, ReverseTime, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, SoftmaxWithLossMultiLabel, Split, TanH, Threshold, Tile, Transpose, WindowData)
*** Check failure stack trace: ***
    @     0x7f4028755daa  (unknown)
I have built warpctc at <home>,and makefile.config also set :

# Whatever else you find you need goes here.
INCLUDE_DIRS := $(PYTHON_INCLUDE) /usr/local/include /usr/include/hdf5/serial /home/**/warp-ctc/build /home/css/warp-ctc/include
LIBRARY_DIRS := $(PYTHON_LIB) /usr/local/lib /usr/lib /usr/lib/x86_64-linux-gnu/hdf5/serial /home/**/warp-ctc/build

**Comment by kasyoukin** [comment]
it could be different ont you environment，I copied libwarpctc.so to /usr/lib/ and ctc.h ctcpp.h to /usr/include.Any way, You should set up according to your actual environment.

The "unkown layer WarpCTCLoss " issue is my mistake ,i forgot to change the Makefile and Makefile.config to enable USE_WARP_CTC.Now it fixed ,you can checkout the latest version
